### ISSUE TITLE
Add session manager

### ISSUE THREAD

**Issue by gilllesdg** [QupZilla/qupzilla#759]
Add possibility to save and restore a session with several tabs.

Kind regards

**Comment by cdbennett** [comment]
Note that #1664 requests adding a menu item to allow restoring previous session after launch.  This is really valuable to me since otherwise I have encountered the following disappointing situation:

1. I have many tabs open, things I have found and am looking forward to reading tomorrow.
2. Qupzilla crashes.  Aw, man.  (Note v1.8.6 on Windows has been crashing every couple of days for me, and it's usually when it's in the background and I'm not even using it -- I just get the "program has stopped working" message)
3. I am reading my email in Outlook, and I click a link.
4. Qupzilla launches to show the link.
5. Now I have a newly started Qupzilla showing the link I clicked.  But my session that I had before it crashed is lost and I can't get it back.

If I start Qupzilla without a URL, just by using the normal program launcher, I will get my old session back since I have the option enabled in Qupzilla preferences.  But if it's launched with a URL then the previous session is lost.
